### ISSUE TITLE
mitmproxy crashes when changing window size

### ISSUE THREAD

**Issue by revintec** [mitmproxy/mitmproxy#806]
I'm using the latest version(0.13) on OS X, inside iTerm2, zsh
to trigger this bug: open mitmproxy, then change the window size.
(put mouse at the bottom-right edge to change width and height at the same time)
do it several times, mitmproxy crash with 100% probability:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/lib/python2.7/site-packages/libmproxy/console/__init__.py", line 499, in run
    self.loop.run()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 278, in run
    self._run()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 375, in _run
    self.event_loop.run()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 678, in run
    self._loop()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 706, in _loop
    self._entering_idle()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 667, in _entering_idle
    callback()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 560, in entering_idle
    self.draw_screen()
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/main_loop.py", line 575, in draw_screen
    self.screen.draw_screen(self.screen_size, canvas)
  File "/usr/local/Cellar/mitmproxy/0.13/libexec/vendor/lib/python2.7/site-packages/urwid/raw_display.py", line 749, in draw_screen
    if osb and osb[y] == row:
IndexError: list index out of range

mitmproxy has crashed!
Please lodge a bug report at:
	https://github.com/mitmproxy/mitmproxy
Shutting down...
```

**Comment by mhils** [comment]
Thanks for the report! Which urwid version are you using (run `pip freeze`)?

**Comment by tunz** [comment]
I had the same issue when urwid version was 1.3.0. It's now fixed after upgrading urwid to 1.3.1.

**Comment by mhils** [comment]
Thanks @tunz! http://urwid.org/changelog.html says that's indeed the case, `pip install --upgrade urwid` should fix this issue.
